### PR TITLE
 [Enterprise 4.8] new procedure for obfuscating insights operator data CCXDEV-4142

### PR DESCRIPTION
**Pull request by xJustin**
From this issue: https://issues.redhat.com/browse/CCXDEV-4142 

@Sergey1011010 @radekvokal can you please take a look.

### CODE REVIEW

**Comment by Sergey1011010** [comment]
looks great, thanks

**Comment by tremes** [comment]
I think it's missing information about resarting the IO pod right @Sergey1011010

**Comment by Sergey1011010** [comment]
@tremes ah, yeah, of course, we also need to mention that the pod needs to be restarted after changing the secret

**Comment by xJustin** [comment]
@Sergey1011010 can you post the command for restarting the IO pod?

**Comment by Sergey1011010** [comment]
@xJustin If using a web console(to be consistent), the user needs to go to workloads -> pods -> choose openshift-insights project -> find insights-operator-* pod and click delete pod from the kebab menu. OpenShift will create a new version of the pod.

File: modules/insights-operator-enable-ofuscation.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * support/insights-operator/disabling-insights-operator.adoc
+
```
Comment (ahardin-rh) [inline modules/insights-operator-enable-ofuscation.adoc:4]: support/insights-operator/disabling-insights-operator.adoc should be support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc

File: modules/insights-operator-enable-ofuscation.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+[id="insights-operator-enable-ofuscation_{context}"]
+= Enabling Insights Operator data obfuscation
+
+
+[role="_abstract"]
```
Comment (ahardin-rh) [inline modules/insights-operator-enable-ofuscation.adoc:11]: We need to remove this "abstract" definition for the time being, as we are not yet on Pantheon 2.

File: support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc [hunk 3]
```diff
@@ -7,6 +7,9 @@ toc::[]
 
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 You may choose to opt out of reporting health and usage data for your cluster.
+
+Alternatively, you can obfuscate the IPv4 addresses and cluster base domains Insights Operator sends to link:https://cloud.redhat.com[cloud.redhat.com] (this feature is disabled by default). See xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#insights-operator-enable-ofuscation_opting-out-remote-health-reporting[Enabling Insights Operator data obfuscation] for details.
```
Comment (ahardin-rh) [inline support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc:11]: [cloud.redhat.com]. This feature is disabled by default.

**Comment by ahardin-rh** [comment]
Preview build: https://deploy-preview-31546--osdocs.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/opting-out-of-remote-health-reporting.html#insights-operator-enable-ofuscation_opting-out-remote-health-reporting

**Comment by ahardin-rh** [comment]
@xJustin Thank you! Just a few minor comments from me.

File: modules/insights-operator-enable-ofuscation.adoc (hunk 2 above)
Comment (xJustin) [reply modules/insights-operator-enable-ofuscation.adoc]: removed

File: support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc (hunk 3 above)
Comment (xJustin) [reply support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc]: Fixed

**Comment by xJustin** [comment]
@ahardin-rh The issues you pointed out are fixed. Is there anything else holding this back from being merged?

**Comment by xJustin** [comment]
This procedure merged as part of a larger chapter: https://github.com/openshift/openshift-docs/pull/32558 

Closing

**Pull request closed without merging**